components/Plants/ShowAlPlantsThisPlaceList/index.js

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
+import { useOnePlace } from "@/utils/helper";
+import { StyledCardContainer } from "@/components/Styling/Card";
+import ShortListPlants from "../ShortListPlants";
+import HeaderPlantsAtThisPlaceList from "../../Places/HeaderPlantsAtThisPlaceList";
+import { useState } from "react";
+import AllPlantsSortedtList from "../AllPlantsSortedList";
+
+export default function ShowAlPlantsThisPlaceList() {
+  const router = useRouter();
+  const site = router.query;
+  const { data, error, isLoading } = useOnePlace(site.id);
+  const [addPlant, setAddPlant] = useState(false);
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (!site.id) {
+    return null;
+  }
+  return (
+    <>
+      <HeaderPlantsAtThisPlaceList
+        placeData={data}
+        setAddPlant={setAddPlant}
+        addPlant={addPlant}
+      ></HeaderPlantsAtThisPlaceList>
+
+      {!addPlant ? (
+        <StyledCardContainer margintop={"130px"}>
+          {data.plants.map((plant) => (
+            <ShortListPlants
+              key={plant._id}
+              plantId={plant.plantid}
+              placeId={site.id}
+              uniquePlantId={plant._id}
```

Comment:
Why do you want to display the plant id for the user? 🤔

Reply:
plant id is used here to get the name of the plant from the db. 
the unique id is used to identify several plants of the same variety in one place. As planned, it should later be possible to create a diary/newsfeed for each plant. The identification is then done via the unique
id.